src/wovnio/wovnphp/CookieLang.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+
+namespace Wovnio\Wovnphp;
+
+class CookieLang
+{
+    const COOKIE_LANG_NAME = 'wovn_selected_lang';
+    const ONE_YEAR = 31536000;
+
+    private $header;
+    private $store;
+
+    public function __construct($header, $store)
+    {
+        $this->header = $header;
+        $this->store = $store;
+    }
+
+    public function getCookieLang()
+    {
+        $cookies = $this->header->getCookies();
+        if (array_key_exists(CookieLang::COOKIE_LANG_NAME, $cookies)) {
+            return $cookies[CookieLang::COOKIE_LANG_NAME];
+        }
+        return null;
+    }
+
+    public function shouldRedirect()
```

Comment:
Still do not like this being here. Make CookieLang simply be about telling us what the value is of the cookie. How we actually use this cookie and whether the feature is enabled is best left to the code doing the redirection.